### PR TITLE
Adding new rsk_getStorageBytesAt JSON-RPC method

### PR DESCRIPTION
**Pull request by asoto-iov**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Review by Vovchyk — CHANGES_REQUESTED** [review]
we'd probably need to disable this method in [here](https://github.com/rsksmart/rskj/blob/a63d4b8b14fb5a21cb02bad886ce8e69b843d3c9/rskj-core/src/main/resources/reference.conf#L428), as it might be resource intensive in some cases

**Comment by asoto-iov** [comment]
> we'd probably need to disable this method in [here](https://github.com/rsksmart/rskj/blob/a63d4b8b14fb5a21cb02bad886ce8e69b843d3c9/rskj-core/src/main/resources/reference.conf#L428), as it might be resource intensive in some cases

sure, I didn't added as I was testing it locally and then I forgot to do it. Thanks for spotting it!

File: rskj-core/src/main/java/co/rsk/rpc/Web3RskModule.java [hunk 1]
```diff
@@ -47,4 +50,7 @@ default void rsk_flush() {
     }
 
     RskModule getRskModule();
+
+    String rsk_getStorageBytesAt(HexAddressParam address, HexNumberParam storageIdx, BlockRefParam blockRefParam);
```
Comment (casiojapi) [inline rskj-core/src/main/java/co/rsk/rpc/Web3RskModule.java:54]: just to understand, is there a reason why the default keyword isn't declared?

Comment (asoto-iov) [reply rskj-core/src/main/java/co/rsk/rpc/Web3RskModule.java:54]: I'm only declaring the method and in this case, it is not needed to add default implementation. 

default methods are used when you want to implement a default method behavior within an interface to not break compatibility with older implementations. It is mainly used in high-exposure interfaces but here it is not needed.

**Review by Vovchyk — APPROVED** [review]

**Review by casiojapi — APPROVED** [review]

File: rskj-core/src/main/java/co/rsk/rpc/Web3RskModule.java (hunk 1 above)
Comment (casiojapi) [reply rskj-core/src/main/java/co/rsk/rpc/Web3RskModule.java:54]: great, huge thanks angel.

**Pull request merged**